app/tests/src/wallet/account.js

```diff
@@ -39,27 +41,57 @@ const initAccount = async (settings, i) => {
     config,
   })
 
-  accounts.push(
-    Account({
-      config,
-      randomInputSeed,
-      randomChangeSeed,
-      cryptoProvider,
-      blockchainExplorer,
-      accountIndex,
-    })
-  )
+  return Account({
+    config,
+    randomInputSeed,
+    randomChangeSeed,
+    cryptoProvider,
+    blockchainExplorer,
+    accountIndex,
+  })
 }
 
 before(async () => {
-  await Promise.all(accountSettings.map(initAccount))
+  ADALITE_CONFIG.ADALITE_CARDANO_VERSION = 'shelley'
+  ADALITE_CONFIG.ADALITE_NETWORK = 'MAINNET'
+  const mockNet = mockNetwork(ADALITE_CONFIG)
+  mockNet.mockBulkAddressSummaryEndpoint()
+  mockNet.mockGetAccountInfo()
+  mockNet.mockGetStakePools()
+  mockNet.mockGetConversionRates()
+  mockNet.mockUtxoEndpoint()
+  mockNet.mockPoolMeta()
+  mockNet.mockGetAccountState()
+  mockNet.mockAccountDelegationHistory()
+  mockNet.mockAccountStakeRegistrationHistory()
+  mockNet.mockWithdrawalHistory()
+  mockNet.mockRewardHistory()
+  mockNet.mockPoolRecommendation()
+
+  await Promise.all(
+    Object.entries(accountSettings).map(async ([name, setting]) => {
+      accounts[name] = await initAccount(setting)
+      return accounts[name]
```

Comment:
I would not return anything to make it clear we are interested only about the completion of the promise and saving a redundant statement

Reply:
Hmm, you are right, I was of the thought that I have to return the promise for `Promise.all` to be able to await it but surprisingly not. Thnx.